charts/trustify/templates/helpers/_infrastructure.tpl

```diff
@@ -38,17 +38,19 @@ Arguments (dict):
 {{- if eq ( include "trustification.application.metrics.enabled" .root ) "true" }}
 - name: METRICS
   value: "enabled"
+- name: OTEL_METRIC_EXPORT_INTERVAL
+  value: {{ with .root.Values.metrics.otelMetricExportInterval }}{{ . | quote }}{{ end }}
```

Comment:
The with needs to capture the whole block. Otherwise it'll end up with an empty env-var.

Reply:
nice it worked, PR updated thanks again :+1: